docs/syntax.md

```diff
@@ -187,7 +187,7 @@ Returns the expression that follows. The last statement of each branch of a func
 
 ```py
 # Allowed, all branches return
-def max(a, b):
+def max(a: T, b: T) -> T:
```

Comment:
This type is not correct. `a` and `b` can't be any type, they must be native numbers. We can't currently express that with bend's syntax, you can just leave it untyped since it's not relevant to the example